<!doctype html>
<html>
<head>
<meta charset="utf-8">
<title>Research</title>
<link href="css/external_style.css" rel="stylesheet" type="text/css">
	<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    
</head>

<body>
	
	<section class="page-header">
      <h1 class="project-name">Antonio de la Vega de Le√≥n</h1>
      <h2 class="project-tagline">Using computer science to aid chemical research</h2>
      <span style="cursor:pointer" onclick="openNav()" id='menu_button'>&#9776; Menu</span>
      

    
    <nav id ="slide_nav" class='sidenav'>
        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
		<ul style="list-style-type: none;">
		<li><a href="index.html">Home</a></li>
		<li><a href="research.html">Research</a></li>
		<li><a href="publications.html">Publications</a></li>
		<li><a href="outreach.html">Outreach</a></li>
		<li><a href="teaching.html">Teaching</a></li>
		<li><a href="cv.pdf">CV</a></li>
		</ul>
	</nav>    
    
    <script>
function openNav() {
    document.getElementById("slide_nav").style.width = "250px";
}

function closeNav() {
    document.getElementById("slide_nav").style.width = "0";
}
</script>
      
    </section>
	<nav id ="left_nav">
		<ul style="list-style-type: none">
		<li><a href="index.html">Home</a></li>
		<li><a href="research.html">Research</a></li>
		<li><a href="publications.html">Publications</a></li>
		<li><a href="outreach.html">Outreach</a></li>
		<li><a href="teaching.html">Teaching</a></li>
		<li><a href="cv.pdf">CV</a></li>
		</ul>
	</nav>
	
	
	<section class="main-content">
		<h1>Research</h1>
		<h2>Current research</h2>
		I am currently a Marie Curie fellow as part of the European Project: "Diagnostics and Drug Discovery Initiative for Alzheimer's Disease" (<a href="https://sites.google.com/site/d3i4adiapp/">D3i4AD</a>).
		As part of this project, I will design and screen for compounds capable of disrupting the interaction between amyloid-beta oligomers and the prion protein.
		This protein-protein interaction is currently thought to be one of the main events leading to neuronal death in the disease.
		Additionally, I am testing the application of multitask machine learning, including deep neural networks, for supporting phenotypic screeening campaigns.
	
	</section>
	
	<section class="main-content">
		<h2>Previous research</h2>
		<h3>Visualization of chemical space</h3>
		Chemical space is the collection of all possible compounds that are physically possible.
		Although not all possible compounds have been synthesized and characterized, public and private collections of compounds have become very large.
		The large size makes systematic analysis of the structure-activity relationship complicated.
		Tailored visualizations are able to ease the analysis by providing either general or focused view of chemical space.
		A large part of my thesis was dedicated to the design and development of visualization to explore complex, high-dimensional chemical spaces.
		I expanded on the chemical space network concept to generate coordinate-free representations of a coordinate-based chemical space, as well as introducing a layout algorithm to provide a global view character to these networks.
		Chemical space views that provide clear regions where favorable compounds are separated from unfavorable were generated to help in multi-objective compound design.
		
		<img src="images/visualizations.png" style="margin-top: 1rem;">
	
	</section>
	
	<section class="main-content">
		<h3>Matched molecular pairs</h3>
		Matched molecular pairs are compound pairs that share a large commong substructure but differ at a specific site.
		They constitute an intuitive way to represent chemical similarity and have become increasingly important for molecular design.
		I performed as part of my thesis several data mining analysis of matched molecular pairs on public databases such as PubChem and ChEMBL.
		This analysis focused on how the structural modification encoded on these pairs changed physico-chemical properties like activity or ionizability.
		Additionally, machine learning models were used to learn the effect of the chemical modification on activity.
		Finally, I proposed an extension of the concept by taking a retrosynthetic approach during the generation of pairs.
		
		<img src="images/mmps.png" style="margin-top: 1rem;">
	
	
	</section>
	
	<section class="main-content">
		<h3>Molecular modelling</h3>
		Docking of compounds on protein binding pockets provides hypothesis on the binding conformation of molecules that can be used for further optimization.
		I assisted medicinal chemistry groups by performing docking studies on compound series in order to better understand the differences in activity present between the compounds.
		I also worked on the analysis of conformations of macrocycle, complex molecules that posses a large ring.
		Many interesting natural products are macrocycles.
		As part of this analysis we compared traditional conformational sampling methods like LowModeMD against short MD simulations to see which was better at finding active conformations.
		<img src="images/modeling.png" style="margin-top: 1rem;">
	
	
	</section>
	
	<section class="main-content">
		<h3>Bioinformatics</h3>
		During the last year of my undergraduate I was part of the workgroup of Prof. Julian Perera.
		They had obtained partial results of the sequencing of Rhodococcus ruber Strain Chol-4.
		I performed the first partial assembly of the contigs that we received and also characterized the regions where the genes responsible for the degradation of cholesterol were located.
	
	</section>
	
</body>
</html>
